### PR TITLE
Replace ownCloud with Nextcloud in the issue template / Fixes #12

### LINKED ISSUE #12: Issues default text
The default text it opens on new issue contains owncloud references.

Would be more appropriate to show Nextcloud.

### PR DESCRIPTION
**Pull request by MariusBluem**
- ownCloud ▶️ Nextcloud (fixes #12)
- Replaced "other components" link with a link to the GitHub Organization (line 4)

ToDo:
- [ ] https://nextcloud.com/security should be up after merging this 😁 @jospoortvliet

### CODE REVIEW

**Comment by mar1u5** [comment]
cc @LukasReschke for security-page 😉

File: issue_template.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!--
-Thanks for reporting issues back to ownCloud! This is the issue tracker of ownCloud, if you have any support question please check out https://owncloud.org/support
+Thanks for reporting issues back to Nextcloud! This is the issue tracker of Nextcloud, if you have any support question please check out https://nextcloud.com/support
 
-This is the bug tracker for the Server component. Find other components at https://github.com/owncloud/core/blob/master/CONTRIBUTING.md#guidelines
+This is the bug tracker for the Server component. Find other components at https://github.com/nextcloud/
```
Comment (jancborchardt) [inline issue_template.md]: Much easier, nice! :D

File: issue_template.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ Tell us what happens instead
...
+**Nextcloud version:** (see Nextcloud admin page)
 
-**Updated from an older ownCloud or fresh install:**
+**Updated from an older Nextcloud/ownCloud or fresh install:**
 
-**Where did you install ownCloud from:**
+**Where did you install Nextcloud from:**
 
-**Signing status (ownCloud 9.0 and above):**
+**Signing status (Nextcloud 9.0 and above):**
```
Comment (jancborchardt) [inline issue_template.md]: We do not use the dot-schema, it is just »Nextcloud 9« :)

**Comment by jancborchardt** [comment]
Nice job @mar1u5, all good apart from my small comments! :)

File: issue_template.md (hunk 2 above)
Comment (mar1u5) [inline issue_template.md]: We could also remove the 9 completely, because there will be no Nextcloud 8.2, 8.1 and 8.0 without signing status 😉

Comment (jancborchardt) [inline issue_template.md]: That doesn’t matter though as we will start with Nextcloud 9. Can you adjust the 9.0 to 9? :)

**Comment by jancborchardt** [comment]
@LukasReschke will there be a security page soon?

**Comment by MorrisJobke** [comment]
I'm fine with merging this before the security page is there. Thanks for the contribution :+1:

**Comment by LukasReschke** [comment]
> @LukasReschke will there be a security page soon?

Yeah… As soon as we have sorted out some more details on this 😉

**Pull request merged**